### DISCUSSION TITLE
How to test the Interactivity API

### DISCUSSION THREAD

**Discussion by FunkyCss** [WordPress/gutenberg#53042] [Interactivity API]
Hallo I would like to ask if there is any way I can use this as a tester, as a plugin extension mostly like a user tester not for code review.

Please let me know the way if this is of course possible.

**Comment by michalczaplinski** [comment]
Hi! 

> any way I can use this as a tester

I'm gonna assume that by "this" you mean the [Interactivity API](https://make.wordpress.org/core/2023/03/30/proposal-the-interactivity-api-a-better-developer-experience-in-building-interactive-blocks/) 🙂

Sure, you can install it as a plugin by grabbing the zip from the [Releases page](https://github.com/WordPress/block-interactivity-experiments/releases). For example, this is the latest one: [block-interactivity-experiments.zip](https://github.com/WordPress/block-interactivity-experiments/releases/download/0.1.12/block-interactivity-experiments.zip)

You can then install it in your WordPress just like you would do with any other plugin. Take a look at the [WP Movies demo](https://github.com/WordPress/wp-movies-demo/) for examples of how to build interactive blocks with it!

**Comment by luisherranz** [comment]
You can now use `@wordpress/create-block` and the latest Gutenberg plugin to test it out.

More info in the new [Getting Started guide](https://github.com/WordPress/gutenberg/blob/trunk/packages/interactivity/docs/1-getting-started.md).
